test/integration/scheduler/framework_test.go

```diff
@@ -1420,6 +1421,11 @@ func TestCoSchedulingWithPermitPlugin(t *testing.T) {
 
 		perPlugin.reset()
 		cleanupPods(cs, t, []*v1.Pod{waitingPod, signallingPod})
+		_, err = cs.CoreV1().Pods(signallingPod.Namespace).Get(signallingPod.Name, metav1.GetOptions{})
```

Comment:
Do you add this for debugging purpose?

Reply:
If extending timeout still doesn't clean up signallingPod, we should fail the test.

Reply:
If we need to add this check, do we need to add this check for other `cleanupPods` in this file?